packages/block-editor/src/components/block-mover/index.js

```diff
@@ -19,7 +19,7 @@ import { withInstanceId, compose } from '@wordpress/compose';
  */
 import { getBlockMoverDescription } from './mover-description';
 import { leftArrow, rightArrow, upArrow, downArrow, dragHandle } from './icons';
-import { IconDragHandle } from './drag-handle';
```

Comment:
It's [on](https://github.com/WordPress/gutenberg/pull/9569#discussion_r217317703)! :trollface: 

Not a blocker and would definitely go with what you think is better given that I'm not very active lately. However, I still think `IconDragHandle` has a place here to hide _how_ the DnD logic works and to maintain the same level of abstraction than the other icons. I find it more useful and fitting now that we've removed the `onDragStart` / `onDragEnd` boilerplate.

Reply:
I agree that it's not all good or bad no matter what we do here.

Personally, while working on it, I find the indirection unnecessary as I was forced to jump between three components (DragHandle, BlockDraggable and the Mover).

> maintain the same level of abstraction than the other icons

What other icons you're talking about, I personally find it a bit more consistent this way as the Mover is rendering all its buttons in the same way (directly usiung `<IconButton>`)

Reply:
> What other icons you're talking about, I personally find it a bit more consistent this way as the Mover is rendering all its buttons in the same way (directly usiung <IconButton>)

I wish that was the case! But I can't help but feeling that the "children as function" pattern is [still odd](https://github.com/WordPress/gutenberg/pull/9311#issuecomment-417031475) and `IconDragHandle` is a nice way to hide it from readers (until they are ready or need to dig in).

However, my concerns are lessened by the fact that, whatever drag component we use, it is not meant to be widely reusable, so I'm ok with this approach.